### DISCUSSION TITLE
Where are cycling of suggestions on a german keyboard layout?

### DISCUSSION THREAD

**Discussion by Calamari** [community/community#6993] [Other Feature Feedback, Questions, & Ideas] [answered]
I have the problem that I don't find alternative suggestions. I tried almost every keycombination with Alt, Alt+Shift & Alt+Strg, but none change the suggestion. I have a german keyboard. Anyone knows where to access those?
Or at least how to remap the keys? I also did not found a setting, where I can change those to something else.

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
If this is question about VSCode, then please check `editor.action.inlineSuggest.showNext` and `editor.action.inlineSuggest.showPrevious` commands in Code->Preferences->Keyboard Shortcuts

**Reply by Calamari** [reply]
Yes it was. That helped.

That also explained why I could not find them, other plugins have prefixed their keymappings (see GitLens, Markdown, etc), but copilot does not:
![Screenshot_2021-10-29_12-56-44](https://user-images.githubusercontent.com/149719/139423203-cc69fa30-8654-440c-8aad-4b17f50e424e.png)

**Reply by k3b** [reply]
@Krzysztof-Cieslak  you saved my day :-)

I found the correspondig info at 

https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/configuring.md under "editor.action.inlineSuggest.showNext"

For intellij/jetbrains/android-studio 

go to Settings/Keymap and search for "copilot"

see also https://github.com/github/copilot-docs/blob/main/docs/jetbrains/configuring.md

**Comment by frwg** [comment]
I have the same issue, but not with Visual Studio Code, but with JetBrains. What can I do?

**Reply by jzenn** [reply]
Is there any progress on this?

**Reply by dijarbytyci** [reply]
You can change your Keyboard Shortcuts on JetBrains IDE like this:
https://www.jetbrains.com/help/idea/configuring-keyboard-and-mouse-shortcuts.html

**Reply by felixmerz00** [reply]
option + [ doesn't work for me either. But there are two default shortcuts (IntelliJ). The secondary shortcuts work for me:
Show previous completions: control + ,
Show next completions: control + .

**Comment by dalito** [comment]
In VSCode the defaults for a German keyboard are:
- `ALT + ´` for going to the next suggestion.
- `ALT + ß` for going back 
- `ALT + ^` to trigger inline suggestions

**Reply by Vanilagy** [reply]
chad

**Reply by Vanilagy** [reply]
Seems to work only on Windows, btw
